llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -308,6 +308,75 @@ InstructionCost VPRecipeBase::computeCost(ElementCount VF,
   return UI ? Ctx.getLegacyCost(UI, VF) : 0;
 }
 
+void VPPartialReductionRecipe::execute(VPTransformState &State) {
+  State.setDebugLocFrom(getDebugLoc());
+  auto &Builder = State.Builder;
+
+  switch (Opcode) {
```

Comment:
Please remove the redundant switch; there's another which checks the Opcode below.

Reply:
Changed to an assertion which can be edited when we add new partial reduction intrinsics.